pkg/v1/cli/command/core/repo.go

```diff
@@ -191,6 +195,7 @@ var deleteRepoCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
+		log.Successf("Successfully deleted repository %s", repoName)
```

Comment:
same here...start the message with a small letter "successfully" to be consistent with other messages.